### PR TITLE
Add addMediaFromBase64 method to the API docs

### PR DESCRIPTION
**Pull request by introwit**

### CODE REVIEW

File: resources/views/laravel-medialibrary/v6/api/adding-files.md [hunk 1]
```diff
@@ -76,6 +76,23 @@ public function addMediaFromRequest(string $keyName): FileAdder
 public function addMultipleMediaFromRequest(array $keyNames): Collection
 ```
 
+### addMediaFromBase64
+
+```php
+/**
+     * Add a base64 encoded file to the medialibrary.
```
Comment (sebastiandedeyne) [inline resources/views/laravel-medialibrary/v6/api/adding-files.md]: Could you fix the indentation here? :)

File: resources/views/laravel-medialibrary/v6/api/adding-files.md [hunk 2]
```diff
@@ -76,6 +76,23 @@ public function addMediaFromRequest(string $keyName): FileAdder
...
+     *
+     * @throws InvalidBase64Data
+     * @throws \Spatie\MediaLibrary\Exceptions\FileCannotBeAdded
+     *
+     * @return \Spatie\MediaLibrary\FileAdder\FileAdder
+     */
+    public function addMediaFromBase64(string $base64data, ...$allowedMimeTypes): FileAdder
+```
+
 Please note the return type of `addMultipleMediaFromRequest` is a collection of `FileAdder`s. This means you'll have to loop over every `FileAdder` to use any of the middle methods. For example:
```
Comment (sebastiandedeyne) [inline resources/views/laravel-medialibrary/v6/api/adding-files.md]: This line belongs to the previous code block iirc.

**Review by sebastiandedeyne — CHANGES_REQUESTED** [review]

**Comment by introwit** [comment]
Done :)

**Review by sebastiandedeyne — APPROVED** [review]

**Comment by sebastiandedeyne** [comment]
👌 Thanks!

**Pull request merged**